### DISCUSSION TITLE
Adding Tailwindcss Theme

### DISCUSSION THREAD

**Discussion by goksagun** [orchidjs/tom-select#843] [Ideas]
Nowadays Tailwindcss is very popular css framework and tom-select should support it.

**Comment by davidnectarestudio** [comment]
Do you know how can I fix this situation? 

It seems that there is a transparent background on dropdown panel. I'm using Tailwind as main CSS framework and combined with TomSelect CSS files is not working well. So I need to apply some class name into my rendered select node to fix the problem. Any hint or idea? Thanks.

![Captura de pantalla 2025-03-20 a les 16 57 51](https://github.com/user-attachments/assets/6ecf8c14-35c4-4434-bee3-a2d5264d10ce)
